test/mock-vendor.go

```diff
@@ -5,7 +5,5 @@ import "github.com/golang/mock/mockgen/model"
 // This assignment exists solely for the purpose of convincing godep
```

Comment:
We don't use godep anymore, while here could you update this comment to reflect this.